README.rst

```diff
@@ -55,7 +55,7 @@ For more help on using Mesa, check out the following resources:
 * `Email list`_
 * `PyPI`_
 
-.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/intro-tutorial.html
+.. _`Intro to Mesa Tutorial` : http://mesa.readthedocs.org/en/latest/tutorials/intro_tutorial.html
```

Comment:
The link works for me.  Looks like it's redirecting to http://mesa.readthedocs.io/en/latest/tutorials/intro_tutorial.html instead, though.  Does that one work for you?